Cargo.toml

```
@@ -44,6 +44,9 @@ readme = "Readme.md"
 [workspace.metadata.release]
 allow-branch = ["main"]
 
+[workspace.dependencies]
+hax-lib = { version = "0.2", git = "https://github.com/hacspec/hax/" }
```

Let's update this while we're here.

```suggestion
hax-lib = { version = "0.2", git = "https://github.com/cryspen/hax/" }
```